launcher/ui/processlist_unix.cpp

```diff
@@ -127,18 +128,21 @@ ProcDataList processList(const ProcDataList &previous)
     const QStringList procIds = procDir.entryList();
     if (procIds.isEmpty())
         return rc;
-    for (const QString &procId : procIds) {
+
+    std::function<ProcData(QString)> procIdToProcData = [](const QString &procId) {
```

Comment:
don't force type erasure here, i.e. don't spell out `std::function<...>` - instead use `auto`. I'd personally even go as far as make this a free function in an anon namespace (or `static`) to cleanup the code.

Reply:
done